BE/src/main/java/org/team4/airbnb/domain/Address.java

```diff
@@ -0,0 +1,11 @@
+package org.team4.airbnb.domain;
+
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Address {
+
+	private String country;
+	private String city;
+	private String gu;
```

Comment:
명확한 영어 단어를 차용하기 위해 `gu` 대신 `district` 라는 변수 네이밍을 추천합니다.
```suggestion
	private String district;
```

Reply:
네 마땅한 이름이 무엇일까 하다가 `gu`라는 단어를 선택했는데요.  Address를 분리할 경우 `district`를 사용하도록 하겠습니다.